configuration/concepts/adult_hiv_followup.csv

```diff
@@ -0,0 +1,72 @@
+uuid,Members,Answers,Fully specified name:fr,Fully specified name:en,Short name:fr,Short name:en,Data class,Data type,_version:1,_order:1002
+259735eb-9844-4ce1-b87d-e78b9e4846fd,,,Poids (kg),Weight (kg),Poids (kg),Weight (kg),Misc,Numeric,,
+2138c4ef-3a33-4900-86d8-618c29919081,,,Taille (cm),Height (cm),Taille (cm),Height (cm),Misc,Numeric,,
+8d891d49-5ae5-40a0-8fef-918ca6b0009f,,,Circonférence abdominale,Abdominal Circumference,Circonférence abdominale,Abdominal Circumference,Misc,Numeric,,
+464bf4b2-4c88-43b4-963d-49a810d9fd12,,,Tour de hanche,Hip Circumference,Tour de hanche,Hip Circumference,Misc,Numeric,,
+f09b8e3e-c7e0-4b6b-887a-3bc6749eb3db,,,Temperature (C),Temperature (C),Temperature (C),Temperature (C),Misc,Numeric,,
+c36bc411-3f10-11e4-adec-0800271c1b75,,,Pouls,Pulse,Pouls,Pulse,Misc,Numeric,,
+c379aa1d-3f10-11e4-adec-0800271c1b75,,,"
+Diastolique (mm Hg)",Diastolic (mm Hg),"
+Diastolique (mm Hg)",Diastolic (mm Hg),Misc,Numeric,,
+c36e9c8b-3f10-11e4-adec-0800271c1b75,,,Systolique (mm Hg),Systolic (mm Hg),Systolique (mm Hg),Systolic (mm Hg),Misc,Numeric,,
+c44cfed2-3f10-11e4-adec-0800271c1b75,,,En séance,Sitting ,En séance,Sitting ,Misc,N/A,,
+c44d6b8e-3f10-11e4-adec-0800271c1b75,,,Couché,Supine,Couché,Supine,Misc,N/A,,
+c44c3c37-3f10-11e4-adec-0800271c1b75,,c44cfed2-3f10-11e4-adec-0800271c1b75;c44d6b8e-3f10-11e4-adec-0800271c1b75,La posture,Posture,La posture,Posture,Misc,Coded,,
+556738c8-de8e-48b0-9efd-066f2c694ff9,c379aa1d-3f10-11e4-adec-0800271c1b75;c36e9c8b-3f10-11e4-adec-0800271c1b75;c44c3c37-3f10-11e4-adec-0800271c1b75,,Valeurs Tension artérielle,Blood Pressure Values,Valeurs Tension artérielle,Blood Pressure Values,Misc,N/A,,
+57cc5b02-93ee-4101-bf64-af2a3527fe24,,,Bon,Good,Bon,Good,Misc,N/A,,
+482bdb99-9ff7-44f4-b52a-44a16d1e76b1,,,Altéré,Altered,Altéré,Altered,Misc,N/A,,
+b2f650c5-d4be-4ee3-81db-33b7a514306f,,57cc5b02-93ee-4101-bf64-af2a3527fe24;482bdb99-9ff7-44f4-b52a-44a16d1e76b1,Etat,State,Etat,State,Misc,Coded,,
+279583bf-70d4-40b5-82e9-6cb29fbe00b4,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Grossesse,Pregnancy,Grossesse,Pregnancy,Misc,Coded,,

```

Comment:
@rubailly For this concept, pregnancy, I created a new concept on the adult initial form, because the original pregnancy concept was not Coded, and had no answers. What do you think we should do here?

Reply:
I think it makes sense to have two different concepts. The one you created (...which I'm also using in this adult hiv followup form) is a question (...as in "is the patient pregnant?") with a YES and NO answer. The other one should be used for example when the patient is asked the reason of stopping ARVs. One of the reason can be "Pregnancy". This would be a concept of class "Misc" and Datatype "N/A".